--- build/bare.fsbl/Makefile	2020-06-25 08:11:13.708686621 +0200
+++ build/bare.fsbl/Makefile.new	2020-06-25 08:12:32.204688106 +0200
@@ -2,7 +2,7 @@
 
 CC := arm-none-eabi-gcc
 CC_FLAGS := -MMD -MP       -mcpu=cortex-a9 -mfpu=vfpv3 -mfloat-abi=hard 
-CFLAGS := 
+CFLAGS := -DFSBL_DEBUG_INFO
 BSP_FLAGS := -O2 -c
 LN_FLAGS :=  -mcpu=cortex-a9 -mfpu=vfpv3 -mfloat-abi=hard -Wl,-build-id=none -specs=Xilinx.spec  -lrsa -Wl,--start-group,-lxil,-lgcc,-lc,--end-group -Wl,--start-group,-lxilffs,-lxil,-lgcc,-lc,--end-group -Wl,--start-group,-lrsa,-lxil,-lgcc,-lc,--end-group                             -Wl,--gc-sections
 
